jax/_src/lax/lax.py

```diff
@@ -4519,25 +4519,43 @@ def _dilate_shape(shape, dilation):
 def _ceil_divide(x1, x2):
   return -np.floor_divide(np.negative(x1), x2)
 
+
+jax_enum_names = [*[es.name for es in xla_client.PaddingType], 'PYTORCH_SAME']
+JaxPaddingType = enum.Enum(
+    'JaxPaddingType',
+    jax_enum_names,
+    start=0,
+)
+
 def padtype_to_pads(in_shape, window_shape, window_strides, padding):
   """Convert padding string to list of pairs of pad values."""
-  PaddingType = xla_client.PaddingType
-
   if isinstance(padding, str):
-    mapping = {'VALID': PaddingType.VALID, 'SAME': PaddingType.SAME}
+    mapping = {
+        'VALID': JaxPaddingType.VALID,
+        'SAME': JaxPaddingType.SAME,
+        'PYTORCH_SAME': JaxPaddingType.PYTORCH_SAME,
+    }
```

Comment:
This is unnecessary. `mapping[some_string]` is equivalent to `JaxPaddingType[some_string]`. See [`enum` docs](https://docs.python.org/3/library/enum.html#programmatic-access-to-enumeration-members-and-their-attributes)

Reply:
Ok, no problem, I just followed the initial style. Fixed.